src/theme/index.scss

```diff
@@ -1,3 +1,6 @@
+@import url('https://fonts.googleapis.com/css?family=Lato');
+@import url('https://fonts.googleapis.com/css?family=Fjalla+One');
```

Comment:
As the application grows using import is going to be a render blocking call, and might affect pageload.. maybe not right away but we can think of async font loading via js.

Reply:
Yup, google fonts most of times create the "_render blocking call_" issue on slow networks.

Reply:
@nitish24p, @palashmon Totally agree with you. I am planning to use https://github.com/typekit/webfontloader. Any thought?